src/content_mapblock.cpp

```diff
@@ -332,9 +332,8 @@ static void makeSmoothLightedCuboid(MeshCollector *collector, const aabb3f &box,
 	}
 	u16 indices[] = {0,1,2,2,3,0};
 	for (s32 j = 0; j < 24; ++j) {
-		int tileindex = MYMIN(j / 4, tilecount - 1);
-		vertices[j].Color = encode_light_and_color(lights[light_indices[j]],
-			tiles[tileindex].color, light_source);
+		vertices[j].Color = encode_light(lights[light_indices[j]],
+			light_source);
```

Comment:
Did I get it correctly that the whole color code is moved to `MeshCollector::applyTileColors()`?